MovieTime/MovieTime/Info.plist

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CFBundleDevelopmentRegion</key>
+	<string>$(DEVELOPMENT_LANGUAGE)</string>
+	<key>CFBundleExecutable</key>
+	<string>$(EXECUTABLE_NAME)</string>
+	<key>CFBundleIdentifier</key>
+	<string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
+	<key>CFBundleInfoDictionaryVersion</key>
+	<string>6.0</string>
+	<key>CFBundleName</key>
+	<string>$(PRODUCT_NAME)</string>
+	<key>CFBundlePackageType</key>
+	<string>APPL</string>
+	<key>CFBundleShortVersionString</key>
+	<string>0.1</string>
+	<key>CFBundleVersion</key>
+	<string>1</string>
+	<key>LSRequiresIPhoneOS</key>
+	<true/>
+	<key>UILaunchStoryboardName</key>
+	<string>LaunchScreen</string>
+	<key>UIMainStoryboardFile</key>
+	<string>Main</string>
+	<key>UIRequiredDeviceCapabilities</key>
+	<array>
+		<string>armv7</string>
+	</array>
+	<key>UISupportedInterfaceOrientations</key>
+	<array>
+		<string>UIInterfaceOrientationPortrait</string>
+		<string>UIInterfaceOrientationLandscapeLeft</string>
+		<string>UIInterfaceOrientationLandscapeRight</string>
+	</array>
+	<key>UISupportedInterfaceOrientations~ipad</key>
+	<array>
+		<string>UIInterfaceOrientationPortrait</string>
+		<string>UIInterfaceOrientationPortraitUpsideDown</string>
+		<string>UIInterfaceOrientationLandscapeLeft</string>
```

Comment:
your app supports landscape, but the upcoming movies screen seems to have been designed only for portrait. How do you envision extending support for landscape?
![simulator screen shot - iphone 8 plus - 2018-01-15 at 13 28 09](https://user-images.githubusercontent.com/20246085/34949682-f2b503b0-f9f7-11e7-81ea-33edb79d429b.png)

Reply:
Actually there is support for Landscape since I reload the information when transitioning in size (portrait to lanscape or landscape to portrait):

```
override func viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator) {
    collectionView.reloadData()
    super.viewWillTransition(to: size, with: coordinator)
}
```
And the size of each cell is calculated according to the decive orientation:
```
let totalWidth: CGFloat
    if viewModel.isDeviceLandscape {
        totalWidth = collectionView.bounds.height - collectionView.contentInset.left - collectionView.contentInset.right - interItemSpacing
    } else {
        totalWidth = collectionView.bounds.width - collectionView.contentInset.left - collectionView.contentInset.right - interItemSpacing
    }
```

The images have a specific ratio:
```
Height = 1.5 * width
```
This is the reason to have the same calculation (2 cells per line) for both orientations. In the next versions it is possible to improve this and put 3 or 4 cells per line, according to the size of the screen and device.

Reply:
That's what I meant: your app supports landscape, but it doesn't look adequate in that orientation. Indeed more cells per row would be an ideal solution here :)